components/NavBar.js

```diff
@@ -129,58 +147,64 @@ const HamburgerMenu = styled.img`
 const Cross = HamburgerMenu;
 
 const DropDownContentContainer = styled.div`
-  padding: 32px 0 27px 20px;
+  padding: 20px 0 27px 0;
   display: flex;
   flex-direction: column;
   gap: 24px;
 `;
 
-const MenuList = ({ setShowDropdown = null }) => {
+const MenuList = ({ setShowDropdown = () => null }) => {
   return  (<>
-    <LinkText href="#" onClick={() => setShowDropdown && setShowDropdown(false)}>About Us</LinkText>
-    <LinkText href="#" onClick={() => setShowDropdown && setShowDropdown(false)}>Hackathons</LinkText>
-    <LinkText href="#" onClick={() => setShowDropdown && setShowDropdown(false)}>Resources</LinkText>
-    <LinkText href="#" onClick={() => setShowDropdown && setShowDropdown(false)}>FAQ</LinkText>
+    <LinkText href="#" onClick={() => setShowDropdown()}>About Us</LinkText>
+    <LinkText href="#" onClick={() => setShowDropdown()}>Hackathons</LinkText>
+    <LinkText href="#" onClick={() => setShowDropdown()}>Resources</LinkText>
+    <LinkText href="#" onClick={() => setShowDropdown()}>FAQ</LinkText>
   </>);
 }
 
-const NavBar = ({ hiring, hiringLink, livePortalLink }) => {
+const NavBar = ({ hiring, hiringLink = '#', livePortalLink = '#' }) => {
```

Comment:
Following up with a comment that's further below, but should we just pull these from the db in this component?

Reply:
done!